### DISCUSSION TITLE
Important Improvements / Missing Features / Feature Requests

### DISCUSSION THREAD

**Discussion by GVLLIFESTYLE** [baimard/gestion#274] [Ideas]
**1.)** It would be nice if the companies address could not only be 1 line, but it could be seperate lines, so that I could format it like this:
Country
City
Street 
...
Alternative mark down support in the text fields would make alot of sense :)
(legal terms, address,....)

**2.)** Every entry / element should be able to be deleted.
      e.g.: Bills cannot be deleted

**3.)** Discount at offer possible. It should be possible to define a discount in % or amount which is then shown as discount in the offer. The discount should be possible on the whole bill or on each position.

**4.)** Documentation in english or german :)

**5.)** Possibility to change language of fields and exported bills / offers from german to en,...

**6.)** Possibility to create custom ad-hoc bills / offers for individual offers (no standart product from the list included)

**Comment by baimard** [comment]
2) Bills is a reference of Quote, both need to be the same. So you just don't delete products in bills.

3) just create a new product name "Discount" with negative price.

4) :)

5) 6) don't understand this part

**Comment by GVLLIFESTYLE** [comment]
5.)

The fields, such as Price, Product Name, Description, ... in my bill / offer are automatically written in german in the preview and in the pdf. I want to change tha language of an offer or an bill to for example english.

2.) I think you missunderstood. My problem is I have deleted all, but I cannot delete bills now. I want if I create some testelements be able to delete them again.

**Reply by baimard** [reply]
For the 2. Maybe it's a bug I m gone have a look.

**Comment by normen** [comment]
I also don't see any way to delete a bill.

**Reply by les-les** [reply]
for testing its hard if u are not able to delete a bill / invoice 
but in real you are not allowed to delete a bill can get u pretty fast in violation of VAT law ... 

Part 6 is easy ... 
take a Product in the list and set the unit Price to 1 EUR name it cash spend per unit 
so if you spend 15 EUR for that client you take this position an set it to 15 and in the comment its like "costs for usb sticks"
same for hour billing with a different rate  

Part 3 is very interesting. 

in 2 ways for bid/offert/devis
1) discount on each POS in % or abs EUR 
2) discount for the whole but be aware there are some reglementations

And this has nothing to do with 
discount payment / skonto zahlung / paiement d'escompte 

this is a whole other ball game and should be carefully looked ad, 
especially when it comes to how long is the bid valid (valid till) and when is the time when its not allowed anymore for the customer to 
reduce discount / skonto / d'escompte  

So there must be a Time / date same as for the bid / offer / devis 
This bid is valid till : 23.03.2023 10:00am CET
on the invoice: You are allowed to reduce the reduce discount / skonto / d'escompte   800 EUR from the SUM included VAT 
till  23.03.2023 10:00am CET after that we will reclaim to the full amount of the invoice

br
LES
